Modules/Typescript/02_oop_part_1/code/src/Pizza.ts

```diff
@@ -0,0 +1,22 @@
+import { Consumable } from './Consumable';
+
+export class Pizza extends Consumable {
+  numberOfSlices: number;
+  slicesEaten: number;
+
+  constructor(numberOfSlices: number, spoiled: boolean) {
+    super('Pizza', numberOfSlices, 0); // not sure what to pass here, only the name?
```

Comment:
Passing these params is correct

Reply:
for number of slices - I would say yes, but the second param is `weight` which we don't pass :?